modules/apps/transfer/keeper/MBT_README.md

```diff
@@ -46,4 +46,4 @@ To wrap Apalache docker image into an executable you might create the following
 docker run --rm -v $(pwd):/var/apalache apalache/mc $@
 ```
 
-In case of any questions please don't hesitate to contact Andrey Kuprianov (andrey@informal.systems).
+In case of any questions please don't hesitate to contact Andrey Kuprianov (<andrey@informal.systems>).
```

Comment:
just wondering  if we still need these docs in here, not sure about latest state of Apalache?

Reply:
Apalache is still in [active development](https://github.com/informalsystems/apalache/commits/main), so I think it's ok to leave this here.